netcore/NetCore.Test/CacheFactoryUnitTest.cs

```diff
@@ -0,0 +1,97 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+using System;
+using Apache.Geode.NetCore;
```

Comment:
Why does the .NET Core library have the "framework" name in the namespace. We didn't do this with any of the other languages/frameworks. In the others the namespace for most client cases is [org.]apache.geode.client. I think this should be changed to `Apache.Geode.Client`. Yes it collides with the .NET Framework version but these can't be used together in the same processes space so there isn't any confusion or issues with that.

Reply:
I've been leaning that way too.  It will make porting ever-so-slightly easier, if nothing else.

Reply:
It will be very complicated to change after the first non-beta release too, so sooner than later on this one.

Reply:
Done